universitat-freiburg-geschichte.csl

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <style xmlns="http://purl.org/net/xbiblio/csl" class="note" default-locale="de-DE" version="1.0" demote-non-dropping-particle="sort-only">
   <info>
-    <title>Universität Freiburg - Geschichte (German)</title>
+    <title>Albert Ludwigs Universität Freiburg - Geschichte (German)</title>
```

Comment:
agreed -- @glennfed please add the hyphens in the title and fix the indentation per comments in line. You can update your pull request by editing the file here: https://github.com/glennfed/styles/blob/patch-2/universitat-freiburg-geschichte.csl (use the pencil icon to edit).
Let us know when you're done, we don't get notifications for that.